src/libstd/sys/windows/fs2.rs

```diff
@@ -402,11 +402,16 @@ pub fn readlink(p: &Path) -> io::Result<PathBuf> {
 }
 
 pub fn symlink(src: &Path, dst: &Path) -> io::Result<()> {
+    symlink_inner(src, dst, false)
+}
+
+pub fn symlink_inner(src: &Path, dst: &Path, dir: bool) -> io::Result<()> {
     use sys::c::compat::kernel32::CreateSymbolicLinkW;
     let src = to_utf16(src);
     let dst = to_utf16(dst);
+    let flags = if dir { 1 } else { 0 };
```

Comment:
Could you add this constant of `1` to the `sys::c` module with the name `SYMBOLIC_LINK_FLAG_DIRECTORY` as well?